test/Sentry.Unity.Tests/IntegrationTests.cs

```diff
@@ -100,12 +99,33 @@ public IEnumerator BugFarmScene_EventCaptured_IncludesApplicationVersionAsReleas
             Assert.AreEqual(Application.version, testEventCapture.Events.First().Release);
         }
 
+        [UnityTest]
+        public IEnumerator BugFarmScene_EventCaptured_UserNameIsEnvironmentUserNameWithDefaultPii()
```

Comment:
Should this be a unit test instead?

Reply:
I think it makes sense to be an integration test since it's toggling an option and verifying what the event processing pipeline did at the end (what user the event ended up with)